examples/js/plugins/COGParser.js

```diff
@@ -0,0 +1,306 @@
+/* global itowns, THREE */
+
+/**
+ * Select the best overview level (or the final image) to match the
+ * requested extent and pixel width and height.
+ *
+ * @param {Source} source The COGSource
```

Comment:
Add implicit type parameters for correct typechecking:
```suggestion
 * @param {Object} source The COGSource
 * @param {Extent} source.extent
 * @param {GeoTIFFLevel[]} source.levels
 * @param {THREE.Vector2} source.dimensions
```

You could define `GeoTIFFLevel` as:
```js
/**
 * @typedef  {Object} GeoTIFFLevel
 * @property {GeoTIFFImage} image
 * @property {number} width
 * @property {number} height
 * @property {number[]} resolution
 */
```

Reply:
Ok, done !